subprocess/subtree_oom.go

```diff
@@ -63,3 +68,58 @@ func getPidFromLog(line string) (int, bool) {
 	}
 	return pid, true
 }
+
+func ClearOOMHandler() http.HandlerFunc {
```

Comment:
Use the gimlet helpers in your functions, like `gimlet.WriteJSONInternalError` here, and include the error message. The agent doesn't post logs to Splunk, so it's hard to get at them if they're only on the task host.

Reply:
Ah gotcha. Let me know if this is what you were thinking.